resource/src/routes/route.mustache

```diff
@@ -0,0 +1,57 @@
+const { Router } = require('express');
+const {{camelCasePlural}} = require('../controllers/{{snakeCasePlural}}');
+
+const serialize = ({{camelCase}}) => {
+  const data = Object.assign({}, {{camelCase}});
+  {{#isUser}}
+  delete(data.password);
+  {{/isUser}}
+  return data;
+};
+
+const router = module.exports = new Router()
+  .get('', function * (req, res) {
+    const list = yield {{camelCasePlural}}.all(req.query);
+    res.json(list.map(serialize));
+  })
+  .get('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.find(req.params.id);
+    res.json(serialize({{camelCase}}));
+  })
+  .post('', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.create(req.body);
+    res.status(201).json(serialize({{camelCase}}));
+  })
+  .put('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.update(req.params.id, req.body);
```

Comment:
This maintains the behaviour of the previous version, but I think that behaviour was wrong. It was intended to be a `replace` but was mistakenly implemented as an `update` and then never addressed. A `PUT` should be a representation of the entire document, and that's what should go straight into the database. What we're doing here is more like a PATCH except without the semantics.

I think we should definitely change this to use `.save` instead of `.update`.

I think we should consider removing the custom `.update` method from the controller super entirely. It's relatively simple to re-implement should the end user need it. Having it there is like handing someone a sharp knife and saying "Don't cut yourself!" when maybe we shouldn't hand it to them in the first place. Not married to this point, just for consideration.